config/locales/server.en.yml

```diff
@@ -2390,6 +2388,7 @@ en:
     suggest_weekends_in_date_pickers: "Include weekends (Saturday and Sunday) in date picker suggestions (disable this if you use Discourse only on weekdays, Monday through Friday)."
 
     splash_screen: "Displays a temporary loading screen while site assets load"
+    navigation_menu: "Determine which navigation menu to be used for the site."
```

Comment:
@mcwumbly Can you have a look to see if the copy for the site setting is OK?

Reply:
Site navigation menu style (sidebar, dropdown or legacy mode)

Reply:
@tgxworld looks like this change may have gotten lost along the way somewhere.